scripts/src/admin/routes.ts

```diff
@@ -25,286 +32,11 @@ function jsonResponse(func: (body: any, params: any, query: any) => Promise<stri
 function wrapService(func: (params: any, query: any) => Promise<string>): RequestHandler {
 	return async function(req: Request, res: Response) {
 		const content = await func(req.params, req.query);
-		/* tslint:disable:no-trailing-whitespace */
-		const html = `
-<html>
-	<head>
-		<title>Marketplace Admin</title>
-		<style>
-		body {
-			font-family: Arial;
-			background: #ffffff;
-		}
-		td {
-			vertical-align: bottom;
-		}
-		tr:nth-child(even) {background: #FFFFFF}
-		tr:nth-child(odd) {background: #FFEEEE}
-		img {
-			height: 50px;
-		}
-		pre {
-			font-family: "Courier New";
-			background: #eeeeee;
-			padding: 5px;
-			width: 250px;
-			overflow: scroll;
-		}
-		.alert {
-			color: red;
-		}
-		.wide {
-			width: 1250px;
-		}
-		.status_completed a {
-			color: green;
-			font-weight: bold;
-		}
-		.status_failed a {
-			color: red;
-			font-weight: bold;
-		}
-		.status_pending a {
-			color: yellow;
-			font-weight: bold;
-		}
-		.status_opened a {
-			color: purple;
-			font-weight: bold;
-		}
-		input {
-			width: 70px;
-		}
-		#footer {
-			height: 70px;
-			width: 100%;
-		}
-		</style>
-		<style>
-		.flex-row-container {
-		    display: flex;
-		    flex-direction: row;
-		    flex-wrap: nowrap;
-		    justify-content: center;
-		    align-content: stretch;
-		    align-items: center;
-	    }
-
+		const html = (await readFile(path.join(__dirname, "../../src/admin/wrapper.html"), { encoding: "utf8" }))
```

Comment:
maybe extract the file path to be a constant in the top of the file?

Reply:
Can you pls guide me what Request type fits here? Or maybe I need to create one?

Reply:
Moved out of the closure function